website/data/docs-nav-data.json

```diff
@@ -403,16 +403,42 @@
         "path": "api-gateway/consul-api-gateway-install"
       },
       {
-        "title": "Technical Specifications",
-        "path": "api-gateway/tech-specs"
+        "title": "Upgrades",
+        "path": "api-gateway/upgrades"
       },
       {
-        "title": "Common Errors",
-        "path": "api-gateway/common-errors"
```

Comment:
Likely needs a redirect added from `api-gateway/common-errors` to `api-gateway/usage/basic-usage#Common_Error_Messages` (I think that's what the generated Markdown ref should be, would need to confirm)